dashboard/controller.py

```diff
@@ -107,19 +108,33 @@ def control_panel():
     abort(requests.codes.unauthorized)
 
 
-@build_enqueue.route('/start/<pipeline>/<branch>', methods=['POST'])
-def start_buildset(pipeline, branch):
+@build_enqueue.route('/start', methods=['POST'])
+def start_buildset():
     if users.get_current_user().is_admin():
-        service.start_buildset(pipeline, branch)
-        return redirect(url_for('status.show_status', pipename=pipeline))
+        pipename = request.form.get('pipename')
+        branch = request.form.get('branch')
+        stoppable_pipelines = config['build_enqueue']['pipelines']
+        for pipeline in stoppable_pipelines:
+            if pipename in pipeline.keys():
+                if branch in pipeline[pipename]:
+                    service.start_buildset(pipename, branch)
+                    return redirect(url_for('status.show_status',
+                                            pipename=pipename))
```

Comment:
When user is admin, but pipeline&branch is not in stoppable, we should return any error code or page. We also should add test if this works correct.